projects/core/src/model/cms.model.ts

```diff
@@ -151,6 +151,7 @@ export interface CmsNavigationComponent extends CmsComponent {
   notice?: string;
   showLanguageCurrency?: string;
   navigationNode?: CmsNavigationNode;
+  resetMenuOnCollapse?: boolean;
```

Comment:
Let's add some jsdoc comment here to explain what this flag does. Its something we all need to do more in general as its very helpful for a quick reference.

eg.
/**
 * Flag indicates whether to reset the state of menu navigation (ie. collapse all submenus) when the menu is closed.
 **/
resetMenuOnCollapse?: boolean;


Also, is it when the navigation panel is collapsed or closed?